runtime/Stdlib_Symbol.resi

```diff
@@ -0,0 +1,105 @@
+/***
+A built-in object that serves as a namespace for globally-unique identifiers.
+
+Compiles to a regular JavaScript Symbol.
+*/
+
+/**
+Type representing a Symbol.
+*/
+type t
+
+/**
+`make(key)`
+
+Makes a new unique Symbol value.
+
+## Examples
+
+```rescript
+Symbol.make("sym1")->assetEqual("sym1")
+```
+*/
+@val
+external make: string => t = "Symbol"
+
+/**
+`getFor(key)`
+
+Searches for existing registered Symbols in the global Symbol registry with the given key and returns it if found. 
+Otherwise a new Symbol gets created and registered with key.
+
+## Examples
+
+```rescript
+Symbol.getFor("sym1")->assetEqual(Symbol.getFor("sym1"))
+```
+*/
+@val
+external getFor: string => t = "Symbol.for"
```

Comment:
you also forgot to update the interface file, and while we're there, I'd also use `@scope` for these bindings, which is cleaner IMO:
```suggestion
@scope("Symbol")
external getFor: string => option<t> = "for"
```

Reply:
Fixed.